web-components/src/components/buttons/button.stories.tsx

```diff
@@ -1,10 +1,11 @@
 import * as React from 'react';
 import OutlinedButton from 'web-components/lib/components/buttons/OutlinedButton';
 import PrevNextButton from 'web-components/lib/components/buttons/PrevNextButton';
-import ContainedButton from 'web-components/lib/components/buttons/ContainedButton';
```

Comment:
Not sure why we were importing components from lib instead of src?

Reply:
Good question. Possibly to match the import we use in real use cases? A user could take these examples and copy/paste them. Other than that, probably no benefit I can think of.